src/view/container/device/Device.tsx

```diff
@@ -53,17 +54,71 @@ const CPX_TOOLBAR_BUTTONS: Array<{ label: any; image: any }> = [
         label: TOOLBAR_ICON_ID.GPIO,
     },
 ];
-
+interface IState {
+    currentDevice: string | undefined;
```

Comment:
Just having the type string is enough. undefined can be assigned to type string.

Reply:
makes sense, I will use ? annotation instead for that property